solutions/src/test/java/ru/sberbank/school/task08/JacksonSaveGameManagerTest.java

```diff
@@ -0,0 +1,101 @@
+package ru.sberbank.school.task08;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import ru.sberbank.school.task08.state.GameObject;
+import ru.sberbank.school.task08.state.InstantiatableEntity;
+import ru.sberbank.school.task08.state.MapState;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class JacksonSaveGameManagerTest {
+
+    private final static JacksonSaveGameManager manager = new JacksonSaveGameManager("f:/temp");
```

Comment:
&#39;static&#39; modifier out of order with the JLS suggestions.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1945471662>*